src/js/components/AppPageComponent.jsx

```diff
@@ -157,7 +157,16 @@ var AppPageComponent = React.createClass({
     });
   },
 
-  onScaleAppError: function (errorMessage) {
+  onScaleAppError: function (errorMessage, statusCode, instances) {
+    if (statusCode === 409) {
+      var appId = this.state.appId;
+      const dialogId = DialogActions.
+        confirm("App scaling failed.Do you want to try forcing?");
```

Comment:
I would rather not have a "Do you want" question, as it would naturally require a "Yes/No" answer but the confirm dialog only has a "OK / Cancel" choice. 

I suggest the following:

`Failed to scale ${appId}. If you want to stop any current deployment of the app and force a new one to scale it, press the OK button.`

@aldipower what do you think?